.github/workflows/build-eslint-jest.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
       #     key: node_modules_${{ runner.os }}_${{ hashFiles('**/yarn.lock') }}
```

Comment:
Might want to enable the cache here to speed it up as well, in the babel repo they fetch all packages once and share the cache between all jobs

https://github.com/babel/babel/blob/0ca601a86f9c3bf041bf6897d61324ddcc9553aa/.github/workflows/ci.yml#L6-L31

Reply:
👍